src/wechaty/user/message.py

```diff
@@ -381,10 +385,10 @@ async def mention_self(self) -> bool:
         Check if a message is mention self.
         :return:
         """
-        self_id = self.puppet.self_id()
+        self_id = self.wechaty.contact_id
```

Comment:
I saw you have use `xxx.get_wechaty()` before, however, I'd like to prefer `xxx.wechaty.yyy` more because it looks clearer.

Please be consistent.

Reply:
This module is designed very early.  When I translate ts to python, I find a problem: python has no static property.  So, we can't set inner static member as typescript.   https://github.com/wechaty/wechaty/blob/master/src/accessory.ts#L30

If we keep consistent, `xxx.wechaty` is a static member, rather than a property which `get/set` inner static member.

Reply:
I'm ok with either design of them.

Let's keep trying our best to keep consistent.